utils/node_local_test.py

```diff
@@ -2563,6 +2563,12 @@ def test_uns_create(self):
         print(stbuf)
         assert stbuf.st_ino < 100
         print(os.listdir(path))
+        rc = run_daos_cmd(self.conf, ['filesystem', 'memquery', self.dfuse.dir])
+        print(rc)
+        assert rc.returncode == 0
+        rc = run_daos_cmd(self.conf, ['filesystem', 'memquery', self.dfuse.dir], use_json=True)
+        print(rc)
+        assert rc.returncode == 0
```

Comment:
i'd like to see a real test here that actually verifies the collected query stats.